lib/rubocop/cop/mixin/preferred_delimiters.rb

```diff
@@ -35,10 +35,10 @@ def preferred_delimiters
             ensure_valid_preferred_delimiters
 
             if preferred_delimiters_config.key?('default')
-              Hash[PERCENT_LITERAL_TYPES.map do |type|
+              PERCENT_LITERAL_TYPES.map do |type|
                 [type, preferred_delimiters_config[type] ||
                   preferred_delimiters_config['default']]
-              end]
+              end.to_h
```

Comment:
I believe this could be further simplified to avoid the `map`

```ruby
              PERCENT_LITERAL_TYPES.to_h do |type|
                [type, preferred_delimiters_config[type] ||
                  preferred_delimiters_config['default']]
              end
```

Reply:
`to_h` with a block was only added in ruby 2.6, but RuboCop supports 2.4, so we can’t use that.

Reply:
Ah I see. Thanks for explaining that.